docs/source/reference.adoc

```diff
@@ -360,3 +360,75 @@ because generating a new token is an expensive operation. Get the access token i
 curl -H "Content-Type: application/json" -X POST -d '{"refresh_token":"<REFRESH_TOKEN>"}' https://auth.openshift.io/api/token/refresh
 
 As a result, you will get a JSON with an access token which you can use in your test.
+
+== Token validation
+
+=== The JWT contents
+
+The token is likely to contain the following claims.
+
+[source]
+ "acr": "0",
+  "allowed-origins": [
+    "https://auth.openshift.io",
+    "https://openshift.io"
+  ],
+  "approved": true,
+  "aud": "fabric8-online-platform",
+  "auth_time": 1534868165,
+  "azp": "fabric8-online-platform",
+  "email": "myusername@gmail.com",
+  "email_verified": false,
+  "exp": 1537460165,
+  "family_name": "Bose",
+  "given_name": "Shoubhik",
+  "iat": 1534868165,
+  "iss": "https://sso.openshift.io/auth/realms/fabric8",
+  "jti": "4ea61a85-f607-4491-b973-ef6314ae4c23",
+  "name": "Shoubhik Bose",
+  "preferred_username": "myusername",
+  "sub": "3383826c-51e4-401b-9ccd-b898f7e2397d",
+  "typ": "Bearer"
+}
+
+
+`sub` is the identity ID of the user.
+
+`jti` is the unique token identifier.
+
+
+=== Expiry
+Ensure the token hasn't expired by 'inspecting' the JWT and verifying the `exp: 1537460165` claim.
+
+=== Verify the signature
```

Comment:
Move it up. Had kept the expiry on top because I thought that's less expensive to check - and if a token is expired, there's no point doing the more expensive signature verification. :) But anyway, we gotta check both :D